lib/server.js

```diff
@@ -264,7 +264,34 @@ function Client(stream, socket) {
   }
   // end service/auth-related ==================================================
 
+  var unrepliedGlobalRequests = [];
```

Comment:
I'm a bit on the fence about the naming here since technically this array could hold some global requests that have the reply already set but are blocked by earlier unreplied requests. Maybe just `globalRequests`? I don't know ....